vmm/src/memory_manager.rs

```diff
@@ -594,6 +618,33 @@ impl MemoryManager {
         }
     }
 
+    fn mbind(
+        addr: *mut u8,
+        len: u64,
+        mode: u32,
+        nodemask: u64,
```

Comment:
I know this is an internal API that we can tweak in the future, but I'd find it cleaner to only provide a `Vec::<u16>` to this function and then build both the bitmask (or eventually an array of u64 bitmasks...) and `maxnode`. This way we'll only have to change that function if/when we want to go further than 64 nodes.

Reply:
Alright, but if I do that, I'll move the node limitation to more than 64. I know QEMU uses 128, do you want me to stick to that?

Reply:
Or maybe no limitation at all. I'll see what I can do.